spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature 'commenting' do
+
+  before {Photo.create caption: 'sunrise'}
+
+  before(:each) do
+    sign_up_with('test@example.com')
+    post_photo('sunrise')
+    leave_comment('nice photo')
+    Timecop.freeze(Time.local(2015))
```

Comment:
Rails/TimeZone: Do not use `Time.local` without zone. Use one of `Time.zone.local`, `Time.local.current`, `Time.local.in_time_zone`, `Time.local.utc`, `Time.local.getlocal`, `Time.local.iso8601`, `Time.local.jisx0301`, `Time.local.rfc3339`, `Time.local.to_i`, `Time.local.to_f` instead. (https://github.com/bbatsov/rails-style-guide#time, http://danilenko.org/2012/7/6/rails_timezones)